test_orchestrator/devtools/populator/test_product_populator.py

```diff
@@ -0,0 +1,47 @@
+from ipaddress import IPv4Address, IPv6Address
```

Comment:
The `test_orchestrator` folder seems to contain test stuff for acceptance_tests specifically, move them to something like `acceptance_tests/fixtures/...`